data/templates/containers/storage.tmpl

```diff
@@ -2,4 +2,4 @@
 
 [storage]
   driver = "vfs"
-  graphroot = "/config/containers/storage"
+  graphroot = "/var/containers/storage"
```

Comment:
This will not survive an image upgrade

Reply:
For containers that support volume mapping, it does not need to exist

Reply:
After the upgrade is started for the first time, the image will be pulled again, and the persistent storage can be used normally through volume mapping (yes, as long as the user sets the container correctly, this is in line with the best practice of container and microservice Technology)